README.md

```diff
@@ -9,19 +9,19 @@ This daemon runs as root. It provides the ability to export existing RBD images
 1. The daemon is a gRPC server, so the host running the server will need to install gRPC packages:
 
 		$ make setup
-		
+
 2. Modify the config file (default ceph-nvmeof.conf) to reflect the IP/ Port where the server can be reached:
 
 		addr = <IP address at which the client can reach the gateway>
 		port = <port at which the client can reach the gateway>
-	
+
 3. To [enable mTLS](#mtls-configuration-for-testing-purposes) using self signed certificates, edit the config file to set:
 
-		enable_auth = True  # Setting this to False will open an insecure port		
+		enable_auth = True  # Setting this to False will open an insecure port
 
 4. Compile protobuf files for gRPC:
-		
-	    $ make grpc 
+
+	    $ make grpc
```

Comment:
Are all README changes just fix up whitespace?  It would be nice to add an example of building, pushing and running these containers.

Reply:
Added an example of building/pushing and running the container